package.json

```diff
@@ -29,7 +29,9 @@
     "flexboxgrid": "6.3.1",
     "heatmap.js": "2.0.5",
     "isomorphic-fetch": "2.2.1",
+    "lodash": "^4.16.4",
     "material-ui": "0.16.1",
+    "moment-duration-format": "^1.3.0",
```

Comment:
Let me explain why those have been added:

- Lodash: I don't feel like implementing my how flow() to compose my filters. I suggest we use the 'lodash/fp' import that is more geared toward functional programming. (I also use: inRange, sortByIndex, map/filter).
- moment-duration-format: I don't think I'm using it, so I guess I'll remove it. Good catch.
- react-addons-css-transition-group: de facto transition component for React
- react-measure: This is to deal with the resizing of the page. @howardchung suggested I use the Redux store size, but I don't see how that can integrate with the flexgrid sizing. @mike-robertson you are welcome to comment.
- seamless-immutable: okay that is a subject of its own, but this manage immutable Object/Array and make it so deeply nested values can be compared with a simple reference equality check. Having immutable objects in the Redux store is considered a good practice.
- react-addons-perf: That is very helpful to dig down React rendering and find performance issues
- react-perf-tool: I think that expose it to the Chrome extension